src/components/Synthetics/TradeBox/TradeBox.tsx

```diff
@@ -791,7 +791,19 @@ export function TradeBox(p: Props) {
     txnPromise.finally(() => {
       setStage("trade");
     });
-  }, [account, isIncrease, isSwap, isWrapOrUnwrap, openConnectModal, setStage, tradeboxTransactions, subaccount]);
+  }, [
+    account,
+    setStage,
+    isWrapOrUnwrap,
+    isSwap,
+    isIncrease,
+    subaccount,
+    openConnectModal,
+    onSubmitWrapOrUnwrap,
+    onSubmitSwap,
+    onSubmitIncreaseOrder,
+    onSubmitDecreaseOrder,
+  ]);
```

Comment:
Not the bug, but just to avoid unnessessary rerenders, becase previously tradeboxTransactions was a new obj every render